src/fixture.c

```diff
@@ -262,6 +274,11 @@ static void ioFlushAppend(struct io *s, struct append *append)
         goto done;
     }
 
+    if (diskFaultTick(s)) {
```

Comment:
`ioFaultTick` above provides similar functionality. Can you look if it's not too much pain to convert the tests that rely on `ioFaultTick` to simulate a disk failure to `diskFaultTick`?

Reply:
Looking into this now

Reply:
Done!

Reply:
Not really, it would be better to have diskFaultTick and then a separate one to simulate network failures. The unified ioFaultTick is really not great imo and I would be happy to get rid of it.

Reply:
This is implemented